ports/vulkan-memory-allocator-hpp/vcpkg.json

```
@@ -1,12 +1,12 @@
 {
   "name": "vulkan-memory-allocator-hpp",
-  "version": "3.0.1",
-  "port-version": 1,
+  "version": "3.0.1-1",
   "description": "C++ bindings for VulkanMemoryAllocator",
```

```suggestion
  "description": "C++ bindings for VulkanMemoryAllocator",
```

I think we should mark this branch as a development branch even if upstream states this branch. We can not change the port name, but it needs to be stated in the description. Because this is not really the upstream. 🤔
https://github.com/malte-v/VulkanMemoryAllocator-Hpp#unmaintained

---

Hi @FrankXie05. What do you mean when you say "development branch"? Can you elaborate on it? Perhaps recommend what to change here? Also, when you say "upstream", do you refer to the original unmaintained repo or the up to date replacement that this port packages?

---

We should indicate in the description that the `YaaZ/VulkanMemoryAllocator-Hpp` we currently maintain is a development branch rather than a true upstream `malte-v/VulkanMemoryAllocator-Hpp`(Even if it is not in maintenance).